providers/base/units/disk/test-plan.pxu

```diff
@@ -17,7 +17,6 @@ include:
     disk/storage_device_.*                     certification-status=blocker
     benchmarks/disk/hdparm-read_.*
     benchmarks/disk/hdparm-cache-read_.*
-    disk/apste_support_on_.*
```

Comment:
This one should not be removed, this is the automated disk test plan that Devices uses, the test is OK for them as APSTE is a feature on consumer grade devices, while not being a feature on enterprise grade devices.